<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainControlPanel</class>
 <widget class="QWidget" name="MainControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>614</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>184</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>184</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
  background-origin: content;
  margin-top: 2.5ex;
  font-weight: bold;
  font-size: 12px;
  color: rgb(30,30,160);
  background-color: rgb(237,237,237);
  padding: 5px;
  border-radius: 4px;
  border: 1px solid rgb(130,130,130);
}
QGroupBox::title {
  subcontrol-origin: 	margin;
  subcontrol-position: top left;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpToolbox">
     <property name="styleSheet">
      <string notr="true">QToolButton {
	width: 22px;
	height: 22px;
	icon-size: 22px;
	padding: 1px;
}

QToolBar {
	padding: 0px;
}</string>
     </property>
     <property name="title">
      <string>Main Toolbox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpInspector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Cursor Inspector</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">QToolButton {
  icon-size: 22px;
  min-width: 1ex;
  border-top: 1px solid rgb(108, 108, 108);
  border-bottom: 1px solid rgb(108, 108, 108);
  border-left: 1px;
  border-right: 1px;
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.0394089 rgba(245, 245, 245, 255), stop:0.487685 rgba(226, 226, 226, 255), stop:0.502463 rgba(215, 215, 215, 255), stop:1 rgba(245, 245, 245, 255));
  width: 25px;
  height: 25px;
}
QToolButton:hover, QToolButton:checked {
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.0394089 rgba(214, 228, 245, 255), stop:0.487685 rgba(193, 205, 221, 255), stop:0.502463 rgba(182, 193, 207, 255), stop:1 rgba(215, 228, 245, 255));
  padding:0px;
  
}
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnCursorInspector">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/root/crosshair.gif</normaloff>:/root/crosshair.gif</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnZoomInspector">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/root/zoom.gif</normaloff>:/root/zoom.gif</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnLabelInspector">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>27</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/root/paintbrush.gif</normaloff>:/root/paintbrush.gif</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDisplayInspector">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>27</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/root/dl_toolbox.png</normaloff>:/root/dl_toolbox.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnSyncInspector">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>27</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/root/network-wireless.png</normaloff>:/root/network-wireless.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnToolInspector">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>27</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/root/tools.png</normaloff>:/root/tools.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stack">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="CursorInspector" name="pageCursorInspector"/>
        <widget class="ZoomInspector" name="pageZoomInspector"/>
        <widget class="LabelInspector" name="pageLabelInspector"/>
        <widget class="DisplayLayoutInspector" name="pageDisplayInspector"/>
        <widget class="SynchronizationInspector" name="pageSyncInspector"/>
        <widget class="QWidget" name="pageToolInspector">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackToolPage">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="SnakeToolROIPanel" name="pageSnakeTool"/>
            <widget class="PaintbrushToolPanel" name="pagePaintbrushTool"/>
            <widget class="QWidget" name="pageBlank"/>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionCrosshair">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/root/crosshair.gif</normaloff>:/root/crosshair.gif</iconset>
   </property>
   <property name="text">
    <string>Crosshair Mode</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Crosshair Mode&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Click and drag the left mouse button to move the 3D cursor in the image. Use the right mouse button to zoom, and the middle mouse button to pan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionZoomPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/root/zoom.gif</normaloff>:/root/zoom.gif</iconset>
   </property>
   <property name="text">
    <string>Zoom/Pan Mode</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Zoom/Pan Mode&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Click and drag the left mouse button to pan the slice views. Use the right mouse button to zoom in and out.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/root/poly.gif</normaloff>:/root/poly.gif</iconset>
   </property>
   <property name="text">
    <string>Polygon Drawing Mode</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Polygon Mode&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Primary tool for manual segmentation. Use the mouse to draw polygons on the 2D slice views, then edit the polygons and fill them with segmentation labels.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionPaintbrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/root/paintbrush.gif</normaloff>:/root/paintbrush.gif</iconset>
   </property>
   <property name="text">
    <string>Paintbrush Mode</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Paintbrush Mode&lt;/span&gt;&lt;/p&gt;&lt;p&gt;A tool for manual editing of segmentations. Hold the left mouse button and drag the mouse to draw with the currently selected label using one of the available brush shapes. Use the right mouse button to draw wuth the background label.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSnake">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/root/snake.gif</normaloff>:/root/snake.gif</iconset>
   </property>
   <property name="text">
    <string>Active Contour Segmentation Mode</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Active Contour Segmentation (Snake) Mode&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Select a region of interest (ROI) for automatic segmentation. Use the mouse to move the boundaries of the ROI, then click the &amp;quot;Segment 3D&amp;quot; button to begin automatic segmentation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CursorInspector</class>
   <extends>QWidget</extends>
   <header>CursorInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZoomInspector</class>
   <extends>QWidget</extends>
   <header>ZoomInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelInspector</class>
   <extends>QWidget</extends>
   <header>LabelInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SnakeToolROIPanel</class>
   <extends>QWidget</extends>
   <header>SnakeToolROIPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PaintbrushToolPanel</class>
   <extends>QWidget</extends>
   <header>PaintbrushToolPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DisplayLayoutInspector</class>
   <extends>QWidget</extends>
   <header>DisplayLayoutInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SynchronizationInspector</class>
   <extends>QWidget</extends>
   <header>SynchronizationInspector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
